### PR TITLE
#[UIC-998] AES encryption of  users confidential detail

### PR DESCRIPTION
**Pull request by jitendra-kumawat**
Task  UIC-998

Did AES encryption of  users confidential details before sending to server

Expected behaviour after this fix:--
-- user's password is shown encrypted  in login request
-- able to login/show error as per  authenticate

### CODE REVIEW

File: superset/templates/superset/base.html [hunk 1]
```diff
@@ -38,4 +38,7 @@
     {% with filename="preamble" %}
       {% include "superset/partials/_script_tag.html" %}
     {% endwith %}
+    {% with filename="login" %}
+      {% include "superset/partials/_script_tag.html" %}
+    {% endwith %}
```
Comment (bipinsoniguavus) [inline superset/templates/superset/base.html]: Can we combine this condition with `preamble` or it doesn't support any syntax to combine?

File: superset/security.py [hunk 2]
```diff
@@ -133,6 +139,39 @@ def _search_ldap(self, ldap, con, username):
             if not user[0][0]:
                 return None
         return user
+    
+    def decryptMessage(self, message):
+        if message :
+            _e = base64.b64decode(message)
+            _i = _e[:BS]
+            _a = AES.new(SK, AES.MODE_CBC, _i)
+            _d = _a.decrypt(_e[BS:]).decode('utf-8')
```
Comment (bipinsoniguavus) [inline superset/security.py]: we should some better variable names.

File: superset/security.py [hunk 3]
```diff
@@ -27,6 +27,12 @@
 from superset.connectors.connector_registry import ConnectorRegistry
 from superset.exceptions import SupersetSecurityException
 
+from Crypto.Cipher import AES
+import base64
+
+BS = 16
+SK = b"qw34sd78fh67asb1"
```
Comment (bipinsoniguavus) [inline superset/security.py]: we should use some better variable names

**Comment by bipinsoniguavus** [comment]
@jitendra-kumawat Please comment on all conversations what are we not doing and what we agreed upon. Also, Please comment if we are fixing lint on this PR else will create a separate PR to take care of lint for all the files present.

File: superset/templates/superset/base.html (hunk 1 above)
Comment (jitendra-kumawat) [reply superset/templates/superset/base.html]: superset/partials/_script_tag.html  file using  js_manifest(filename)  function and to load multiple files we can use common name  like 'entry'  but  for two distinct files we need to add them separately

File: superset/security.py (hunk 2 above)
Comment (jitendra-kumawat) [reply superset/security.py]: because of security ,using short named variable

File: superset/security.py (hunk 3 above)
Comment (jitendra-kumawat) [reply superset/security.py]: because of security ,using short named variable

**Pull request merged**